### PR TITLE
Clarify chaff should be on both endpoints

### PR DESCRIPTION
**Pull request by sethvargo**
/assign @mikehelmick 

**Release Note**

```release-note
Clarify chaff should be on both endpoints
```

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -558,7 +559,8 @@ will not be processed). The server will respond with a fake response that your
 client **MUST NOT** process or parse. The response will not be a valid JSON
 object.
 
-Client's should sporadically issue chaff requests to mirror real-world usage.
+Client's should sporadically issue chaff requests to mirror real-world usage
+for both the `/verify` and `/certificate` endpoints.
```
Comment (mikehelmick) [inline docs/api.md:563]: add key server publish as well

Comment (sethvargo) [reply docs/api.md]: Done

File: docs/api.md [hunk 2]
```diff
@@ -558,7 +560,8 @@ will not be processed). The server will respond with a fake response that your
 client **MUST NOT** process or parse. The response will not be a valid JSON
 object.
 
-Client's should sporadically issue chaff requests to mirror real-world usage.
+Client's should sporadically issue chaff requests to mirror real-world usage
+for both the `/verify`, `/certificate`, and key server stats endpoints.
```
Comment (mikehelmick) [inline docs/api.md:564]: key server publish endpoints, not stats.

Comment (sethvargo) [reply docs/api.md]: Done

**Review by mikehelmick — APPROVED** [review]

**Pull request merged**